src/app/components/ConnectorTypeCard/ConnectorTypeCard.tsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import { FunctionComponent } from 'react';
+import { useTranslation } from 'react-i18next';
+
+import {
+  Card,
+  CardHeader,
+  CardActions,
+  Label,
+  CardTitle,
+  Popover,
+  CardBody,
+  DescriptionList,
+  DescriptionListGroup,
+  DescriptionListTerm,
+  DescriptionListDescription,
+} from '@patternfly/react-core';
+import {
+  BuildIcon,
+  BuilderImageIcon,
+  OutlinedQuestionCircleIcon,
+} from '@patternfly/react-icons';
+
+import {
+  ObjectReference,
+  ConnectorTypeAllOf,
+} from '@rhoas/connector-management-sdk';
+
+export type ConnectorTypeCardProps = {
+  connector: ObjectReference | ConnectorTypeAllOf;
```

Comment:
I'd suggest having this component just accept the strings/fields it needs to show, then it won't be coupled to the connector object.  And you can then also simplify the typing I think.

Reply:
yes, that could be another way. I will update it.

Reply:
done